odxtools/cli/_print_utils.py

```diff
@@ -153,7 +153,7 @@ def extract_parameter_tabulation_data(parameters: List[Parameter]) -> Dict[str,
 
     for param in parameters:
         name.append(param.short_name)
-        byte.append(param.byte_position)
+        byte.append(getattr(param, "byte_position", None))
```

Comment:
using getattr breaks type checking, and penalize performance

Reply:
I buy the type checking argument, but performance wise it should be basically the same because in python classes are implemented as glorified dictionaries.

How can this be fixed alternatively? according to the ODX spec we cannot rely on the fact that all parameters have a `byte_position` attribute (even though in practice we can).